sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -1245,6 +1245,39 @@ def test_idxmax(self):
     self._run_test(lambda s2: s2.idxmax(), s2)
     self._run_test(lambda s2: s2.idxmax(skipna=False), s2)
 
+  def test_pipe(self):
+    def df_times(df, column, times):
+      df[column] = df[column] * times
+      return df
+
+    def df_times_shuffled(column, times, df):
+      return df_times(df, column, times)
+
+    def s_times(s, times):
+      return s * times
+
+    def s_times_shuffled(times, s):
+      return s_times(s, times)
+
+    df = pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6]}, index=[0, 1, 2])
+    s = pd.Series([1, 2, 3, 4, 5], index=[0, 1, 2, 3, 4])
+    # s2 = pd.Series([6, 7], index=[5, 6])
+
+    func_1 = df_times
+    func_2 = s_times
+    # func_3 = frames.DeferredSeries.append #pd.Series.append
+    func_4 = df_times_shuffled
+    func_5 = s_times_shuffled
+
+    self._run_inplace_test(lambda df: df.pipe(func_1, 'A', 2), df)
+    self._run_inplace_test(lambda df: df.pipe((func_4, 'df'), 'A', 2), df)
+
+    self._run_test(lambda s: s.pipe(func_2, 2), s)
+    self._run_test(lambda s: s.pipe((func_5, 's'), 2), s)
```

Comment:
nit: could you remove the comments, and just refer to the functions directly rather than aliasing as `func_x`?